api/v1/helper.go

```diff
@@ -30,7 +30,8 @@ const (
 
 const invalidVfIndex = -1
 
-var MANIFESTS_PATH = "./bindata/manifests/cni-config"
+var SRIOV_NET_ATTACH_DEF_PATH = "./bindata/manifests/cni-config/sriov"
+var OVN_NET_ATTACH_DEF_PATH = "./bindata/manifests/cni-config/ovn"
```

Comment:
can you rename this to ovnkube? there are other cni that used ovn (e.g https://github.com/alauda/kube-ovn)  which we may want to add in the future

Reply:
Done